common/presenters/move-to-card-component.js

```diff
@@ -29,6 +30,20 @@ function moveToCardComponent({
 
     const moveMetaItems = []
 
+    if (showIdentifierNumber && profile && profile.person && move.to_location) {
+      if (move.to_location.location_type === 'prison') {
```

Comment:
Should we ensure that we have a value for prison number here? Otherwise we could end up with just the label.

Thinking about it, it might be neater to remove the `profile` checks from the outer statement as check for the value here:
```suggestion
      if (move.to_location.location_type === 'prison' && profile?.person?.prison_number) {
```

But double check with @elizabethgoddard what she'd prefer to happen when there is no value. Either not sure the meta item or some fallback text like "No prison number".